.travis/apisix_cli_test.sh

```diff
@@ -447,17 +447,17 @@ git checkout conf/config.yaml
 
 make init
 
-count=`grep "ssl_session_tickets off;" conf/nginx.conf || true`
-if [ $count -ne 1 ]; then
+count=`grep "ssl_session_tickets off;" conf/nginx.conf | wc -l `
```

Comment:
we can use this way: https://github.com/apache/apisix/blob/8ed24711a6c659fa8040310a24e86091d104a699/.travis/apisix_cli_test.sh#L122

Reply:
done. `grep` -> `grep -c`